Sources/Down/AST/Visitors/AttributedStringVisitor.swift

```diff
@@ -53,7 +59,12 @@ extension AttributedStringVisitor: Visitor {
     }
 
     public func visit(list node: List) -> NSMutableAttributedString {
-        listPrefixGenerators.append(ListItemPrefixGenerator(list: node))
+
+        listPrefixGenerators.append(
+            listPrefixGeneratorBuilder.build(
+            listType: node.listType,
+            numberOfItems: node.numberOfItems,
+            nestDepth: node.nestDepth))
```

Comment:
If you create a convenience init that accepts the list node (and possibly use a closure as mentioned above), you could simplify this to:

```swift
listPrefixGenerators.append(listPrefixGeneratorBuilder(node))
```